recipe/meta.yaml

```diff
@@ -46,8 +46,15 @@ requirements:
 test:
   imports:
     - batman
+  source_files:
+    - test_cases/
+    - batman/
+  requires:
+    - pytest-runner
+    - mock
   commands:
     - batman -h
+    - py.test test_cases/ --ignore=test_cases/Mascaret
```

Comment:
I am not in favor of spending CI ressources for this. CI is already done in the repo.

Reply:
Ok, but we need to test at least once on windows

Reply:
For matplotlib, usually the we enable the agg backend during tests so no X11 is required.

Reply:
Indeed, that's what we do on gitlab.

About windows, why do you want windows support? I am not sure we will ever have windows user... But if there is a real need on your side, we can think about doing this on gitlab directly. Otherwise, we will end up with patches every now and then. And I really do not like these patches. As the name suggest, this is not the tagged version.

Reply:
Just thought it should be possible to enable it as every dependency is there.